rules/windows/execution_command_shell_started_by_unusual_process.toml

```diff
@@ -26,9 +26,31 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where host.os.type == "windows" and event.type == "start" and
-  process.name : "cmd.exe" and
-  process.parent.name : ("lsass.exe",
+sequence with maxspan=3m 
+[process where host.os.type == "windows" and event.type == "start" and ( ( process.name: "dllhost.exe" and not process.args: ("/Processid:{CA8C87C1-929D-45BA-94DB-EF8E6CB346AD}") ) or ( process.name: ("lsass.exe",
```

Comment:
you don't need the `sequence` to add this exclusion. If you just leave the original logic, you can append it with

```sql
  ... and
  not (process.parent.name : "dllhost.exe" and process.parent.args : "/Processid:{CA8C87C1-929D-45BA-94DB-EF8E6CB346AD}")
```

Reply:
ah ok - I didn't think the process.parent.args was available...let me try that

Reply:
Thanks for your tip - not sure why I assumed process.parent.args was not in available!